packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -13,6 +13,9 @@ export type FieldContainerProps = ThemedComponentBaseProps<{
     error: typeof FormHelperText;
     warning: typeof FormHelperText;
     helperText: typeof FormHelperText;
+    secondaryHelperText: typeof FormHelperText;
+    helperTextWrapper: "div";
+    helperTextContainer: "span";
```

Comment:
The naming of the two elements is a bit confusing, wrapper and container seem to be the same thing...

Maybe `helperTextsWrapper` for the element wrapping all the helper-text**s** and `helperTextContent` for the text-content of the individual helper-text 🤔 

```suggestion
    helperTextsWrapper: "div";
    helperTextContent: "span";
```